tools/eslint-plugin-cdk/README.md

```diff
@@ -0,0 +1,9 @@
+# eslint-plugin-cdk
+
+Eslint plugin for the CDK repository. Contains rules that need to be applied specific to the CDK repository.
+
+## Rules
+
+* `no-core-construct`: Forbid the use of `Construct` and `IConstruct` from the "@aws-cdk/core" module.
+  Instead use `Construct` and `IConstruct` from the "constructs" module.
+  Rule only applies to typescript files under the `test/` folder.
```

Comment:
Is the fact that this is running only against `test` encoded into the rule or just the configuration on the usage site?

Reply:
Encoded into the rule, but can be extracted into the configuration.

Reply:
Since this is a temporary rule, I am okay with leaving it like this. Please make sure we have a task in your board to delete this linter plugin. It feels like something we shouldn't be in the business of maintaining in the long run.